### PR TITLE
Update path and region tag

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Intended to update this page: https://cloud.google.com/vpc/docs/using-firewalls#terraform

### CODE REVIEW

File: vpc/basic_firewall_rule/main.tf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2022 Google LLC
```
Comment (rsamborski) [inline vpc/basic_firewall_rule/main.tf:2]: If this is a new sample, please update year to 2023.

File: vpc/basic_firewall_rule/main.tf [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+# [START compute_firewall_create]
+resource "google_compute_firewall" "rules" {
+  name        = "my-firewall-rule"
+  network     = "default"
+  description = "Creates firewall rule targeting tagged instances"
+
+  allow {
+    protocol = "tcp"
+    ports    = ["80", "8080", "1000-2000"]
```
Comment (rsamborski) [inline vpc/basic_firewall_rule/main.tf:25]: Can we match the behaviour of other code samples ([example](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/5741faad622a6dbbb39e3c76526b17b32890be68/compute/client_library/snippets/firewall/create.py#L101))? We should open only ports 80 and 443.

File: vpc/basic_firewall_rule/main.tf [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  source_tags = ["foo"]
```
Comment (rsamborski) [inline vpc/basic_firewall_rule/main.tf:28]: To match other samples can we open traffic from all internet ([see here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/5741faad622a6dbbb39e3c76526b17b32890be68/compute/client_library/snippets/firewall/create.py#L104))?

**Review by rsamborski — CHANGES_REQUESTED** [review]

File: vpc/basic_firewall_rule/main.tf (hunk 1 above)
Comment (msampathkumar) [reply vpc/basic_firewall_rule/main.tf:2]: This is an old file from 2022 as mentioned at https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/firewall_with_target_tags/main.tf

Comment (rsamborski) [reply vpc/basic_firewall_rule/main.tf:2]: Make sense.

**Review by rsamborski — APPROVED** [review]

**Pull request merged**